### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by Bludidi**
## In this pull request I have implemented the following:

- Create a class Classroom with @Label instance variable that should be initialized in the constructor and add Setter and getter for @Label
- Create the has-many/belongs-to relationship between Classroom and Student.
- Create a class Book with @title and https://github.com/author instance variables that should be initialized in the constructor and add Setters and getters for instance variables.
- Create a class Rental with @Date instance variable that should be initialized in the constructor and add Setter and getter for @Date
- Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental.

### CODE REVIEW

File: Relationship/book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative '../Relationship/rental'
+
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```
Comment (mikethreels) [inline Relationship/book.rb]: - [ ] attr_accessor for rentals should be set as well.

File: Entities/person.rb [hunk 2]
```diff
@@ -1,15 +1,17 @@
 require_relative '../Decorator/nameable'
+require_relative '../Relationship/rental'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (mikethreels) [inline Entities/person.rb]: - [ ] attr_accessor for rentals should be set as well.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Brian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @Bludidi,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Set up associations correctly 👍🏽
✅ Good commit messages 👍🏽
✅ Good descriptive PR 👍🏽

## Optional Changes

- [ ] Kindly add a description, steps to clone your repo, and other related content to your project in the README. The previous reviewer did not highlight them. Please update your README to have relevant content to your app 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**